### ISSUE TITLE
Broken as of 0.106

### ISSUE THREAD

**Issue by DrSpaldo** [ofekashery/vertical-stack-in-card#75]
It appears the below breaking change has broken this card.

https://developers.home-assistant.io/blog/2020/02/18/106-custom-card-changes/

Any help would be good :)

**Comment by bieniu** [comment]
It works for me with HA 0.106.0b2.

![image](https://user-images.githubusercontent.com/478555/75117311-8a9b9780-5670-11ea-9e2a-806383b41197.png)

**Comment by DrSpaldo** [comment]
Yeah, so it is working for me with everything, except one card. As soon as I put that card in with custom:vertical-stack-in-card then I get a bunch of errors.  This is the offending card/code:

```
      - type: vertical-stack
        cards:
        - type: picture-glance
          title: Harmony Upstairs
          entities:
          - switch.samsung_tv
          - switch.kodi
          state_image:
            "PowerOff": /local/img/power_off_red.png
            "TV": /local/img/samsung_logo.png
            "HTPC": /local/img/kodi_logo.png
          entity: sensor.harmony_upstairs_activity
        - type: conditional
          conditions:
          - entity: switch.samsung_tv
            state: 'on'
          card: !include upstairs_samsung.yaml
        - type: conditional
          conditions:
          - entity: switch.kodi
            state: 'on'
          card: !include upstairs_kodi.yaml
```
As soon as I change the above card to vertical-stack then the others all work with custom:vertical-stack-in-card

The strange part about it, is that I have two different harmony cards using pretty much the same code, one for upstairs and one for downstairs using picture-glance and conditional options. The downstairs one is working fine but the upstairs one breaks it as soon as I use custom:vertical-stack-in-card

Very strange?

**Comment by vajonam** [comment]
I think its got to do with the conditional, because I have a similar card that causes issues with conditional view. others that dont have conditional types work okay.

**Comment by iantrich** [comment]
lazy loading was reverted for 0.106. It will break again in 0.107. https://developers.home-assistant.io/blog/2020/02/18/106-custom-card-changes

**Comment by brgerig** [comment]
I confirm that this is broken in 0.107. Any card types used in this card that aren't used elsewhere in the view are broken.

**Comment by starkillerOG** [comment]
same problem in 0.107

**Comment by niccolobusato** [comment]
also here on 0.107.0

**Comment by i00** [comment]
Worked find in 0.106.X ... so don't think this is really the place to post about 0.107.X ... yes it doesn't work though :(

**Comment by SeLLeRoNe** [comment]
It worked on 0.106.X because the lazy load has been reverted in one of the 0.106.x releases.
As @iantrich menthioned that change that is impacting this card has been re-introduced into 0.107.0 (https://developers.home-assistant.io/blog/2020/02/18/106-custom-card-changes/) therefore this was originally happening on 106 but because of HA change it is now happening from 107, basically the issue is the same.

**Comment by i00** [comment]
Nope.. does work for me in 107 .. was wrong before... just the resource didn't get automatically transferred over to the new config :)

**Comment by SeLLeRoNe** [comment]
Well it does not work for me because of that change Ian menthioned.
In fact, every non-custom card that is loaded under this card, doesn't work.
Example:
![image](https://user-images.githubusercontent.com/9599441/77066309-31aaef00-69e3-11ea-97c0-3ff4f2db4759.png)

**Comment by GoNzCiD** [comment]
Afer upgrade to 0.107.1, it's broken for me too. 
I vertical card doesn't work with map and markdown. For the map I get this error:
![imagen](https://user-images.githubusercontent.com/12912439/77067743-c7477e00-69e5-11ea-9ba6-6153976b6f76.png)


The lovelace code:
```yaml
      - cards:
          - content: "hello"
            type: markdown
          - aspect_ratio: '16:9'
            entities:
              - entity: device_tracker.1
              - entity: device_tracker.2
              - entity: device_tracker.3
              - entity: device_tracker.4
              - entity: device_tracker.5
            type: map
        type: custom:vertical-stack-in-card
```

**Comment by McGiverGim** [comment]
I have errors in several cards too, with 0.107.0 and 0.107.1.

**Comment by alemuro** [comment]
Hey all, I had the same error after updating to `0.107.x` from `0.104`.
 
There is a workaround to fix it. You can use "lovelace card preloader" plugin, in order to preload the cards that are unable to load. I was in trouble with the `conditional` lovelace card, now It is fixed :) 

More info here: https://community.home-assistant.io/t/lovelace-card-preloader/176655

**Comment by pplucky** [comment]
> Hey all, I had the same error after updating to `0.107.x` from `0.104`.
>
Same here, from 0.106.5 to 0.107.1

> There is a workaround to fix it. You can use "lovelace card preloader" plugin, in order to preload the cards that are unable to load. I was in trouble with the `conditional` lovelace card, now It is fixed :)
>
It works for most cards used within vertical-stack, just not divider, it still shows:
>
`Custom element doesn't exist: hui-divider-row.`

**Comment by SeLLeRoNe** [comment]
To clarify, all the custom cards will still work, only the "standard" HA cards are not.
It doesn't matter which version are you updating from, it is a change done from the 0.107.0 onwards that requires a change in the plugin.

**Comment by SeLLeRoNe** [comment]
Thanks for the tip @alemuro

**Comment by i00** [comment]
I only get that error when I load stuff in an entity row .. working fine in the vertical-stack-in-card
... either way I think this maybe fix your issues:
https://github.com/gadgetchnnel/lovelace-card-preloader/

**Comment by RomRider** [comment]
I've created a new card which is almost a drop-in replacement for vertical-stack-in-card and a bunch of other features:
https://community.home-assistant.io/t/stack-in-card-drop-in-replacement-for-vertical-stack-in-card/180072

**Comment by ofekashery** [comment]
Fixed in version [0.3.0](https://github.com/ofekashery/vertical-stack-in-card/releases/tag/v0.3.0), thanks!
